python/mxnet/gluon/nn/activations.py

```diff
@@ -181,6 +181,25 @@ def hybrid_forward(self, F, x):
         return F.LeakyReLU(x, act_type='selu', name='fwd')
 
 
+class CELU(HybridBlock):
+    r"""
+    Continuous Exponential Linear Unit (CELU)
+        https://arxiv.org/abs/1704.07483
```

Comment:
might help to add documentation on what it is than just linking a paper
see the existing doc here for [LeakyReLU](http://mxnet.incubator.apache.org/api/python/ndarray/ndarray.html#mxnet.ndarray.LeakyReLU)

Reply:
i will do that. But I will do it more on these lines - http://mxnet.incubator.apache.org/api/python/gluon/nn.html#mxnet.gluon.nn.LeakyReLU as this a gluon block rather than an ndarray function.